src/core/service.c

```diff
@@ -294,6 +294,7 @@ usec_t service_restart_usec_next(Service *s) {
 
         if (n_restarts_next <= 1 ||
             s->restart_steps == 0 ||
+            s->restart_usec <= 0 ||
```

Comment:
Doesn't matter, but I find `== 0` more readable for unsigned values

Reply:
I have no preference about that, but Lennart prefers `<= 0` even for unsigned variables, iirc.